### DISCUSSION TITLE
How to test a spoke scene?

### DISCUSSION THREAD

**Discussion by yajo** [mozilla/hubs#3449] [Spoke Questions] [answered]
I think my scene is ready but I'd rather test it before publishing to Hubs. But do I really have to hit *Publish to Hubs...* to test it? If I want to modify it, can I do it later?

**Comment by cvanmeurs** [comment, marked as answer]
It won't be shared with anyone when you publish.  You sorta have to publish it to test it.  The URL will be unique to you and it won't be shared with anyone unless you happen to give someone the link.  It's only when you check the box that says "Allow Mozilla to promote my scene" that we will even bother to look at the scene ourselves (and you don't even ever have to do that at all - only if you want to share it with the community).

And yes, you can modify it later.  The URL will remain the same every time you publish it again, too.

**Reply by yajo** [reply]
Great! It works exactly as I'd expect it to. Thank you very much.

**Reply by cvanmeurs** [reply]
You're welcome!  Good luck!  :)
